recipes/rwa-python/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "0.8.5" %}
+
+package:
+  name: rwa-python
+  version: {{ version }}
+
+source:
+  url: https://github.com/DecBayComp/RWA-python/archive/v{{ version }}.tar.gz
+  sha256: 9bcc928ccfb22d1353d59a95c98062ef2b6543c56044ee92cf52ded32cb6d262
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install ."
+
+requirements:
+  host:
+    - h5py
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
+    - six
```

Most of these probably aren't needed in host
```suggestion
    - pip
    - python
```